ReleaseHistory.md

```diff
@@ -16,6 +16,7 @@
 - NEW => new feature 
 
 ## UNRELEASED
```

Comment:
>UNRELEASED [](http://example.com/codeflow?start=3&length=10)

the notes in this section and in 4.1.0 are not ordered properly.

DEP always comes first for example. Please refer to the release notes maintenance guide to identify the proper ordering (and add that information to the doc if it isn't there.

See 4.0.0 for the correct ordering.

Reply:
Reordered the notes in this and the section below